PlaywrightSolar/Tests/CatalogFiltersTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using PlaywrightSolar.PageObjects;
+
+namespace PlaywrightSolar.Tests;
+
+internal class CatalogFiltersTest : UiTestFixture
+{
+    private SolarShopPage _solarShopPage;
+    private SolarPanelsPage _solarPanelsPage;
+    
+    [SetUp]
+    public void SetupSolarTechnologyShopPage()
+    {
+        _solarShopPage = new SolarShopPage(Page);
+        _solarPanelsPage = new SolarPanelsPage(Page);
+    }
+    
+    [Test]
+    public async Task VerifyCatalogFilteringByManufacturer()
+    {
+        // Arrange
+        await _solarPanelsPage.GoToSolarPanelsPage();
+        await _solarPanelsPage.SolarPanelsPageOpened();
+        await _solarPanelsPage.OpenProductFilter();
+        
+        // Act
+        // Possible values: "Abi-Solar", "C&T Solar", "JA Solar", "Jinko Solar", "SOLA", "Ulica Solar", "Yingli Solar"
+        var productsNameBeforeFiltering = await _solarPanelsPage.GetInitialProductsName();
+        var productsNameAfterFiltering = await _solarPanelsPage.GetProductsNameByManufacturer("SOLA");
```

Comment:
due to hidden implementation this test case looks like:
Open page
Open filters
Get all names
Get all names by particular manufacturer
Compare that they are different (of course they are different because these looks like 2 different lists selected by 2 different criteria)